### ISSUE TITLE
git rm -r . failure under windows

### ISSUE THREAD

**Issue by samccone** [shinnn/gulp-gh-pages#85]
**platform**: windows
**package version**: 0.5.4

---

originally reported here https://github.com/PolymerElements/polymer-starter-kit/issues/678

---
##### Log:

```
[22:33:37] Starting 'deploy-gh-pages'...
[22:33:40] [gh-pages] Cloning repo
[22:33:40] [gh-pages] Checkout branch `gh-pages`
[22:33:40] [gh-pages] Updating repository
[22:33:45] 'deploy-gh-pages' errored after 8.64 s
[22:33:45] Error in plugin 'gulp-gh-pages'
Message:
    Command failed: C:\Windows\system32\cmd.exe /s /c "git rm -r ."
fatal: pathspec '.' did not match any files

Details:
    killed: false
    code: 128
    signal: null
    cmd: C:\Windows\system32\cmd.exe /s /c "git rm -r ."
[22:33:45] 'build-deploy-gh-pages' errored after 51 s
[22:33:45] Error in plugin 'run-sequence'
Message:
    An error occured in task 'deploy-gh-pages'.
```

---

Seems somewhat related to

shinnn/gulp-gh-pages#13
shinnn/gulp-gh-pages#14
shinnn/gulp-gh-pages#61

**Comment by vonWolfehaus** [comment]
Ah, good I'm not the only one. Any help on this? Any way I can help?

**Comment by woneob** [comment]
I have the same problem as this issue.

- OS - Wndows 10 
- git version 2.7.0.windows.1

**Comment by DanielRamosAcosta** [comment]
Same here

  * Ubuntu 15.10
  * git - 2.5.0
  * node 5.6.0

**Comment by zoxon** [comment]
Me too
```sh
Message:
    Command failed: C:\Windows\system32\cmd.exe /s /c "git rm -r ."
error: the following file has local modifications:
    assets/favicons/manifest.json
(use --cached to keep the file, or -f to force removal)

Details:
    killed: false
    code: 1
    signal: null
    cmd: C:\Windows\system32\cmd.exe /s /c "git rm -r ."
```
* gulp-gh-pages 0.5.4
* Windows 7 x64
* Git 2.1.4
* npm 2.15.5
* node 4.4.5

**Comment by brunogarciacosta** [comment]
Creating a random dummy file inside of the 'gh-pages' (just so "git rm -r ." could fetch a file) branch fixed it for me.

**Comment by nguyenlamzx** [comment]
@brunogarciacosta yes, it worked for me

**Comment by shinnn** [comment]
Fixed in [v0.6.0-4](https://github.com/shinnn/gulp-gh-pages/tree/v0.6.0-4). Thanks for the report, @samccone!
